source/fundamentals/connection/tls.txt

```diff
@@ -74,8 +74,11 @@ Configure Certificates
 Java applications that initiate TLS/SSL requests require access to
 cryptographic certificates that prove identity for the application
 itself as well as other applications with which the application
-interacts. To configure access to these certificates in your application,
-you should use the JVM Trust Store and the JVM Key Store.
+interacts. You can configure access to these certificates in your application with
+two mechanisms:
+
+- The JVM Trust Store and JVM Key Store
```

Comment:
Question: Are these two co-dependent? If so, should each one have its own bullet?

Reply:
Interestingly, it seems like the answer is [yes and no](https://stackoverflow.com/questions/54897842/java-sslcontext-init-with-null-key-manager) (according to this Stack Overflow post which may or may not be good information), depends on your JDK. I think "are JVM Trust Store and JVM Keystore Co-dependent?" is worth asking in technical review, as TLS certificate management is something I don't know well, and I don't think it is worth looking into too much for this PR specifically (should definitely learn it well soon however).

I think this breakdown is alright (this mirrors the breakdown on the old java doc as well), where specifying Key/Trust with `SSLContext.init` is one section and JVM Key and JVM Trust are their own sections which are discussed together in an intro paragraph.